src/common/utils/getHoldingLocations.js

```diff
@@ -0,0 +1,63 @@
+import {
+  batchRequest,
+  HOLDINGS_API,
+  LIMIT_MAX,
+  LOCATIONS_API,
+} from '@folio/stripes-acq-components';
+
+import { extendKyWithTenant } from './utils';
+
+const DEFAULT_DATA = [];
+
+export const getHoldingLocations = async ({ ky, searchParams, signal, tenantId }) => {
+  const holdings = await ky
```

Comment:
why don't you use react query?

Reply:
I have used here https://github.com/folio-org/ui-receiving/pull/574/files#diff-c647abf1842f671ea2c2ef6ef784933f56d88529097c0fce466fdde930a44299R44-R45